completion/tests/test_services.py

```diff
@@ -89,7 +154,7 @@ class CompletionDelayTestCase(CompletionSetUpMixin, TestCase):
     """
 
     @ddt.data(1, 1000, 0)
-    def test_get_completion_by_viewing_delay_ms(self, delay):
+    def test_get_complete_on_view_delay_ms(self, delay):
```

Comment:
Have you made sure you caught all uses of this method in edx-platform?

Reply:
Yup. They are used in two views and there is is bokchoy coverage for both.